exp/internal/controllers/machinepool_controller_phases.go

```diff
@@ -33,6 +33,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/cluster-api/cmd/clusterctl/client/tree"
```

Comment:
we shouldn't be importing clusterctl here, see my comment above

Reply:
fixed